sim/humanoid_gym/play_h5.py

```diff
@@ -0,0 +1,213 @@
+    policy = ppo_runner.get_inference_policy(device=env.device)
+
+    # Export policy if needed
+    EXPORT_POLICY = True
+    if EXPORT_POLICY:
+        path = os.path.join(".")
+        export_policy_as_jit(ppo_runner.alg.actor_critic, path)
+        print("Exported policy as jit script to: ", path)
+
+    # Prepare for logging
+    env_logger = Logger(env.dt)
+    robot_index = 0
+    joint_index = 1
+    stop_state_log = 1000
+
+    # Initialize HDF5 file
+    now = datetime.now()
+    now = now.strftime("%Y-%m-%d_%H-%M-%S")
+    h5_file = h5py.File("data" + now + ".h5", "w")
+
+    # Create dataset for actions
+    max_timesteps = stop_state_log
+    num_dof = env.num_dof
+    dset_actions = h5_file.create_dataset("actions", (max_timesteps, num_dof), dtype=np.float32)
+
+    # Create dataset of observations
+    buf_len = len(env.obs_history)  # length of observation buffer
+    dset_2D_command = h5_file.create_dataset("observations/2D_command", (max_timesteps, buf_len, 2), dtype=np.float32)
+    dset_3D_command = h5_file.create_dataset("observations/3D_command", (max_timesteps, buf_len, 3), dtype=np.float32)
+    dset_q = h5_file.create_dataset("observations/q", (max_timesteps, buf_len, num_dof), dtype=np.float32)
+    dset_dq = h5_file.create_dataset("observations/dq", (max_timesteps, buf_len, num_dof), dtype=np.float32)
+    dset_obs_actions = h5_file.create_dataset(
+        "observations/actions", (max_timesteps, buf_len, num_dof), dtype=np.float32
+    )
+    dset_ang_vel = h5_file.create_dataset("observations/ang_vel", (max_timesteps, buf_len, 3), dtype=np.float32)
+    dset_euler = h5_file.create_dataset("observations/euler", (max_timesteps, buf_len, 3), dtype=np.float32)
+
+    if RENDER:
+        camera_properties = gymapi.CameraProperties()
+        camera_properties.width = 1920
+        camera_properties.height = 1080
+        h1 = env.gym.create_camera_sensor(env.envs[0], camera_properties)
+        camera_offset = gymapi.Vec3(3, -3, 1)
+        camera_rotation = gymapi.Quat.from_axis_angle(gymapi.Vec3(-0.3, 0.2, 1), np.deg2rad(135))
+        actor_handle = env.gym.get_actor_handle(env.envs[0], 0)
+        body_handle = env.gym.get_actor_rigid_body_handle(env.envs[0], actor_handle, 0)
+        logger.info("body_handle: %s", body_handle)
+        logger.info("actor_handle: %s", actor_handle)
+        env.gym.attach_camera_to_body(
+            h1, env.envs[0], body_handle, gymapi.Transform(camera_offset, camera_rotation), gymapi.FOLLOW_POSITION
+        )
+
+        fourcc = cv2.VideoWriter_fourcc(*"MJPG")
+
+        # Creates a directory to store videos.
+        video_dir = run_dir() / "videos"
+        experiment_dir = video_dir / train_cfg.runner.experiment_name
+        experiment_dir.mkdir(parents=True, exist_ok=True)
+
+        dir = os.path.join(experiment_dir, datetime.now().strftime("%b%d_%H-%M-%S") + str(args.run_name) + ".mp4")
```

Comment:
Make the video and h5 file names the same

Reply:
fixed